src/wallet/components/WalletModal.tsx

```diff
@@ -158,9 +158,8 @@ export function WalletModal({
           border.lineDefault,
           border.radius,
           background.default,
-          'w-[323px] p-6 pb-4',
-          'flex flex-col items-center gap-4',
-          'relative',
+          'w-[22rem] p-6 pb-4',
```

Comment:
Does this work at either `w-80` or `w-96` ?

Otherwise, we could add an `88` as `22rem` to the `tailwind.config.ts`

cc @brendan-defi @alessey - thoughts?

Reply:
I suppose` w-80 `or `w-96` could work but I was pairing with Mind and we landed in between at `22rem`.

Reply:
I'm fine hardcoding properties to exact measurements if tailwind doesn't have a class for it

Reply:
i'm creating a size 88 tailwind extension for 22rem for walletisland

Reply:
Sounds good, I'll update this style when that's in